README.md

```diff
@@ -7,17 +7,34 @@ the RV32IMC instruction set architecture.
 
 <p align="center"><img src="doc/images/blockdiagram.svg" width="650"></p>
 
-Ibex offers several configuration parameters to meet the needs of various application scenarios.
-The options include two different choices for the architecture of the multiplier and divider unit,
-as well as the possibility to drop the support for the "M" extension completely. In addition, the
-"E" extension can be enabled when opting for a minimum-area configuration.
-
 This core was initially developed as part of the [PULP platform](https://www.pulp-platform.org)
 under the name "Zero-riscy" \[[1](https://doi.org/10.1109/PATMOS.2017.8106976)\], and has been
 contributed to [lowRISC](https://www.lowrisc.org) who maintains it and develops it further. It is
 under active development, with further code cleanups, feature additions, and test and verification
 planned for the future.
 
+## Configuration
+
+Ibex offers several configuration parameters to meet the needs of various application scenarios.
+The options include different choices for the architecture of the multiplier unit, as well as a range of performance and security features.
+
+| Config | "small" | "maxperf" | "maxperf-pmp-bm"  |
+| ------ | ----------------- | ----------------- | ----------------- |
+| Features | RV32IMC, 3 cycle mult | RV32IMC, 1 cycle mult, Branch target ALU, Writeback stage | RV32IMCB, 1 cycle mult, Branch target ALU, Writeback stage, 16 PMP regions |
+| Performance (Coremark/MHz) | 2.44 | 3.09 | 3.09 |
+| Area - Yosys (kGE) | 33.15 | 39.03 | 63.32 |
+| Area - Commercial (estimated kGE) | 24.00 | 29.00 | 46.00 |
+| Verification status | Green | Amber | Amber |
```

Comment:
We should have some description of what we mean by 'Green' and 'Amber' (and presumably 'Red'), maybe better off in the documentation rather than the READMe